tests/schemas/invalid/test_invalid_use_of_id_relationships_with_type.json

```diff
@@ -0,0 +1,66 @@
+{
+  "mime_type": "application/json/ons/eq",
+  "language": "en",
+  "schema_version": "0.0.1",
+  "data_version": "0.0.3",
+  "survey_id": "0",
+  "title": "Test invalid use of id relationships",
+  "theme": "default",
+  "description": "A questionnaire to test invalid use of id relationships with type",
```

Comment:
I'm not sure it's clear here what "with type" means, we could instead use:
> A questionnaire to test invalid use of block id relationships

The filename should also be updated if this is changed.

Reply:
I think the filename should change to `test_invalid_use_of_block_id_relationships.json`